recipes/py-xxhash/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "py-xxhash" %}
+{% set version = "2.0.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xxhash/xxhash-{{ version }}.tar.gz
+  sha256: 58ca818554c1476fa1456f6cd4b87002e2294f09baf0f81e5a2a4968e62c423c
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+    - xxhash
+  run:
+    - python
+    - xxhash
```

Comment:
Does xxhash have the right ABI requirements and run exports?

Reply:
Nope, I guess `run_exports` should be added `xxhash` instead of fixing it here: https://github.com/conda-forge/xxhash-feedstock/blob/master/recipe/meta.yaml